s3pypi/core.py

```diff
@@ -89,6 +90,19 @@ def parse_distribution(path: Path) -> Distribution:
     return Distribution(name, version, path)
 
 
+def parse_distributions(paths: List[Path]) -> List[Distribution]:
+    dists = []
+    for path_ in paths:
```

Comment:
Is `path_` a leftover from another version of this code that tried to avoid a name clash? Would `path` also work fine here?

Reply:
I didn't want to get it confused with pathlib.Path

Reply:
I can switch it to path

Reply:
Alright, that would be more consistent. `path` is used in other places in this file as well.